apps/parsley/docs/06-Downloading-Logs.md

```diff
@@ -7,67 +7,88 @@ Note that there is a 1 minute timeout on log downloads. If you find that your do
 
 ## Via Spruce
 
 Right click the "Raw" button and select the "Save Link As..." option. This will save the logs as a `.txt` file.
 
-#### Task Logs
+### Task Logs
+
 ![task logs button](./images/task_logs_button.png)
 
-#### Test Logs
+### Test Logs
+
 ![test logs button](./images/test_logs_button.png)
 
-## Via Terminal
+## Via Rest API
 
-Use your terminal to fetch logs from the Evergreen REST API. Note that special characters in the URL may have to be escaped.
+Using our REST API, you can download the logs for a given task or test.
 
-#### Task Logs
-```properties
-# with cURL
-curl -H Api-User:<your_user_id> -H Api-Key:<your_api_key> https://evergreen.mongodb.com/rest/v2/tasks/<task_id>/build/TaskLogs -o output.txt
+### Authentication
+
+To authenticate your requests, please reference [this documentation](https://docs.devprod.prod.corp.mongodb.com/evergreen/API/Authentication).
 
-# with wget
-wget --header="Api-User:<your_user_id>" --header="Api-Key:<your_api_key>" https://evergreen.mongodb.com/rest/v2/tasks/<task_id>/build/TaskLogs -O output.txt
+### Task Logs
+
+[Route documentation](https://docs.devprod.prod.corp.mongodb.com/evergreen/API/REST-V2-Usage#tag/tasks/paths/~1tasks~1%7Btask_id%7D~1build~1TaskLogs/get).
+
+```bash
+# With cURL
+curl -H "Authorization: Bearer $(evergreen client get-oauth-token)" https://evergreen.corp.mongodb.com/rest/v2/tasks/<task_id>/build/TaskLogs -o output.txt
+
+# With wget
+wget --header="Authorization: Bearer $(evergreen client get-oauth-token)" https://evergreen.corp.mongodb.com/rest/v2/tasks/<task_id>/build/TaskLogs -O output.txt
 ```
 
-#### Test Logs
-```properties
-# with cURL
-curl -H Api-User:<your_user_id> -H Api-Key:<your_api_key> https://evergreen.mongodb.com/rest/v2/tasks/<task_id>/build/TestLogs/<test_log_path> -o output.txt
+> Note, your session may be expired. You should run `evergreen login` to refresh your session before running the above command(s).
 
-# with wget
-wget --header="Api-User:<your_user_id>" --header="Api-Key:<your_api_key>" https://evergreen.mongodb.com/rest/v2/tasks/<task_id>/build/TestLogs/<test_log_path> -O output.txt
+### Test Logs
+
+[Route documentation](https://docs.devprod.prod.corp.mongodb.com/evergreen/API/REST-V2-Usage#tag/tasks/paths/~1tasks~1%7Btask_id%7D~1build~1TestLogs~1%7Bpath%7D/get).
+
+```bash
+# With cURL
+curl -H "Authorization: Bearer $(evergreen client get-oauth-token)" https://evergreen.mongodb.com/rest/v2/tasks/<task_id>/build/TestLogs/<test_log_path> -o output.txt
+
+# With wget
+wget --header="Authorization: Bearer $(evergreen client get-oauth-token)" https://evergreen.mongodb.com/rest/v2/tasks/<task_id>/build/TestLogs/<test_log_path> -O output.txt
 ```
 
+> Note, your session may be expired. You should run `evergreen login` to refresh your session before running the above command(s).
```

Comment:
This command is coming from [this PR](https://github.com/evergreen-ci/evergreen/pull/9519). I'm not merging this PR until that PR is merged. The command is needed for scripts that are still user-driven